spec/features/admin/action_creation_spec.rb

```diff
@@ -152,15 +138,21 @@ def fill_in_basic_info(title:, summary:, description:)
     fill_in "Title", with: title
     fill_in_editor "#action_page_summary", with: summary
     fill_in_editor "#action_page_description", with: description
-    fill_in_select2 "#action_page_category_id", with: category.title
+    select(category.title, from: "action_page_category_id") #we should use the label to find the select but the label here is not correctly pointing to the select
   end
 
   def next_section
     click_on "Next"
     sleep 0.05
   end
 
+  def skip_banner_selection
+    expect(page).to have_selector("#images", visible: true, wait: 5)
```

Comment:
Doing an expect like this is a better way to make sure a page is settled before calling actions that will error if an element isnt visible. Even though I'm waiting for potentially 5 seconds it will automatically continue as soon as it resolves true. You also know that the element isnt available which is a more useful error than what shows if it cant fill in an input.

Reply:
oh, that's a great pattern!!